ci-operator/step-registry/troubleshooting-panel/tests/ui/troubleshooting-panel-tests-ui-commands.sh

```
@@ -0,0 +1,98 @@
+  "import sys,json; v=json.load(sys.stdin)['openshiftVersion']; print('.'.join(v.split('.')[:2]))" 2>/dev/null || echo "")
+
+export KUBECONFIG=/tmp/kubeconfig
+export CYPRESS_BASE_URL="https://${console_route}"
+export CYPRESS_LOGIN_IDP=kube:admin
+export CYPRESS_LOGIN_USERS="kubeadmin:${kubeadmin_password}"
+export CYPRESS_OPENSHIFT_VERSION="${ocp_version}"
+export ARTIFACT_DIR="${ARTIFACT_DIR:-/tmp}"
+export CYPRESS_CACHE_FOLDER=/tmp/Cypress
+export NO_COLOR=1
+
+cd "${repo_dir}/web"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)